sklearn/base.py

```diff
@@ -292,6 +293,24 @@ def __repr__(self):
         return '%s(%s)' % (class_name, _pprint(self.get_params(deep=False),
                                                offset=len(class_name),),)
 
+    def __getstate__(self):
+        if type(self).__module__.startswith('sklearn.'):
+            return dict(self.__dict__.items(), __version__=__version__)
+        else:
+            return dict(self.__dict__.items())
```

Comment:
> Can we still have that only on models inside scikit-learn? It would be a class
> attribute,

+1: see in the class of BaseEstimator. It should be very simple, just
adding

<pre>

   _version_ = sklearn.__version__

</pre>

before the __init__ in the class.

The only doubt that I have is about circular imports by accessing the
central sklearn module.